pages/docs/reference/data-science-overview.md

```diff
@@ -7,137 +7,44 @@ title: "Kotlin for Data Science"
 
 # Kotlin for Data Science
 
-![Kotlin for data science]({{ url_for('asset', path='images/landing/data-science/data-science-overview.png')}})
+From building ETL pipelines to productionizing machine learning models, Kotlin can be a great choice for 
```

Comment:
Starting from specific terms may scare newcomers who just wondering about Kotlin and looking at all overview pages. 
Maybe start from some simple commonly known things?

Reply:
Fixed in 0589f3228c213d9072011459e408d442132c4108